src/libpgmoneta/status.c

```diff
@@ -373,13 +371,11 @@ pgmoneta_status_details(SSL* ssl, int client_fd, bool offline, uint8_t compressi
          workspace_size = 0;
       }
 
-      if (strlen(config->common.servers[i].hot_standby) > 0)
-      {
-         hot_standby_size = pgmoneta_directory_size(config->common.servers[i].hot_standby);
-      }
-      else
+      hot_standby_size = 0;
+      for (int j = 0; j < config->common.servers[i].hot_standby_count; ++j)   // TODO: details?
```

Comment:
What details ?

Reply:
I left a comment to check if there are more to add or not.

I cleaned it now